### ISSUE TITLE
I want to build Classic XA 13.2.2.0 locally using macOS, but it keeps failing

### ISSUE THREAD

**Issue by maonaoda** [dotnet/android#9253]
### Android framework version

Other

### Affected platform version

None

### Description

```
=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
Downloading Mono archive
=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=

Checking if all runtime files are present
  • some files are missing, download and extraction required
Downloading Mono archive from https://xamjenkinsartifact.azureedge.net/mono-sdks/android-debug-Darwin-d9a6e8710b37cd7f16cf52eff4f772556e57cc41.7z
Failed to obtain Mono archive size. HTTP status code: BadRequest (400). Mono runtimes will be rebuilt
Downloading Mono archive from https://xamjenkinsartifact.azureedge.net/mono-sdks/android-debug-Darwin-d9a6e8710b37cd7f16cf52eff4f772556e57cc41.7z
Failed to obtain Mono archive size. HTTP status code: BadRequest (400). Mono runtimes will be rebuilt
Downloading Mono archive from https://xamjenkinsartifact.azureedge.net/mono-sdks/android-debug-Darwin-d9a6e8710b37cd7f16cf52eff4f772556e57cc41.7z
Failed to obtain Mono archive size. HTTP status code: BadRequest (400). Mono runtimes will be rebuilt
Mono archive not present

=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
Installing Mono runtimes
=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=


Step Xamarin.Android.Prepare.Step_InstallMonoRuntimes failed: Unable to build mono runtimes from sources.
System.InvalidOperationException: Step Xamarin.Android.Prepare.Step_InstallMonoRuntimes failed: Unable to build mono runtimes from sources.
 ---> System.NotImplementedException: Unable to build mono runtimes from sources.
   at Xamarin.Android.Prepare.Step_InstallMonoRuntimes.Execute(Context context) in /Users/wangyk/Projects/Practice/xamarin-android/build-tools/xaprepare/xaprepare/Steps/Step_InstallMonoRuntimes.cs:line 34
   at Xamarin.Android.Prepare.Step.Run(Context context) in /Users/wangyk/Projects/Practice/xamarin-android/build-tools/xaprepare/xaprepare/Application/Step.cs:line 42
   at Xamarin.Android.Prepare.Scenario.Run(Context context, Log log) in /Users/wangyk/Projects/Practice/xamarin-android/build-tools/xaprepare/xaprepare/Application/Scenario.cs:line 37
   --- End of inner exception stack trace ---
   at Xamarin.Android.Prepare.Scenario.Run(Context context, Log log) in /Users/wangyk/Projects/Practice/xamarin-android/build-tools/xaprepare/xaprepare/Application/Scenario.cs:line 48
   at Xamarin.Android.Prepare.Context.Execute() in /Users/wangyk/Projects/Practice/xamarin-android/build-tools/xaprepare/xaprepare/Application/Context.cs:line 845
   at Xamarin.Android.Prepare.App.Run(String[] args) in /Users/wangyk/Projects/Practice/xamarin-android/build-tools/xaprepare/xaprepare/Main.cs:line 174
```

### Steps to Reproduce

make prepare MSBUILD=msbuild

**Comment by maonaoda** [comment]
https://github.com/dotnet/android/issues/9048
It that reason ?

**Comment by jonpryor** [comment]
Fixed in #9255.

Please use the [d17-5 branch](https://github.com/dotnet/android/commits/d17-5) to get this fix.
